keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifact/ArtifactListenerTests.kt

```diff
@@ -33,7 +34,7 @@ internal class ArtifactListenerTests : JUnit5Minutests {
     .provenance("https://my.jenkins.master/jobs/fnord-release/58")
     .build()
 
-  val debianArtifact = DebianArtifact(name = "fnord", deliveryConfigName = "fnord-config")
+  val debianArtifact = DebianArtifact(name = "fnord", deliveryConfigName = "fnord-config", vmOptions = VirtualMachineOptions(baseOs = "bionic", regions = setOf("us-west-2")))
```

Comment:
There are a lot of places where I'm having to specify this. Not sure if there's a sensible way to default it and I don't feel like it should be nullable.

Reply:
I think it's sensible to default it, can we default from properties in the config file? it's going to be pretty installation specific I think.

Reply:
+1. Not unlike what we already do with base images.

Reply:
Yeah, that might work. I don't think we can default the regions, but ultimately I'd like that  to go away anyway (hopefully later in this PR, maybe in a future one).